### ISSUE TITLE
Invisible play button in QGIS 3.4.4 and QGIS 3.6.0 

### ISSUE THREAD

**Issue by jwirges** [anitagraser/TimeManager#291]
I encountered problems running the time manager plugin in QGIS 3.4.4. and QGIS 3.6.0:
-The play button in the control panel was not visible. But it was possible to click the "empty space" at that position.
-After registering layers and closing the menu, these entries were lost. Reopening the menu showed an empty list of layers. I tried several times but it was not possible to register the layers.

I work in Windows 10 on a Lenovo Thinkpad.
There were no problems with the Time Manager Plugin in my previously installed version QGIS 2.18.14. I used the same data here, and it worked.

**Comment by anitagraser** [comment]
I can reproduce the disappearing play button but not the disappearing layers.

**Comment by anitagraser** [comment]
Please open a separate ticket with test data and steps to reproduce the layer registration issue

**Comment by anitagraser** [comment]
I've published a new version https://plugins.qgis.org/plugins/timemanager/version/3.1.1/

**Comment by jwirges** [comment]
Thank you.
I prepared subsets of my dataset (selection of features, selection of columns), but there the registration worked. Testing around I was not able to pinpoint the problem.
A version of the dataset I used in QGIS 3.6. seems to not register because it contains data entries of the form 1980.01.01 along with faulty entries of the form .01.01. 
In QGIS 3.4.4. I seem to have a project file(?) where no layers can be added in the Time Manager. Opening a new project resolves the issue. As I am not an able to pinpoint the problem I would like to drop this issue.
